<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalOptionsDialog</class>
 <widget class="QDialog" name="GlobalOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Global options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="SetGEOCODE_checkBox">
        <property name="text">
         <string>set GEOCODE range</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="SetCampaignName_checkBox">
        <property name="text">
         <string>if unknown set campaign name</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Select parameters dialog</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="ShowShortName_checkBox">
        <property name="text">
         <string>show parameter short name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="ShowMethod_checkBox">
        <property name="text">
         <string>show method</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="ShowComment_checkBox">
        <property name="text">
         <string>show comment</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Date/Time format</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QRadioButton" name="realISO_radioButton">
        <property name="text">
         <string>YYYY-MM-DD'T'HH:mm (real ISO)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="ISO_radioButton">
        <property name="text">
         <string>YYYY-MM-DD HH:mm (better for EXCEL)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="Day_radioButton">
        <property name="text">
         <string>YYYY-MM-DD</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OK_pushButton">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel_pushButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>SetGEOCODE_checkBox</tabstop>
  <tabstop>SetCampaignName_checkBox</tabstop>
  <tabstop>ShowShortName_checkBox</tabstop>
  <tabstop>ShowMethod_checkBox</tabstop>
  <tabstop>ShowComment_checkBox</tabstop>
  <tabstop>realISO_radioButton</tabstop>
  <tabstop>ISO_radioButton</tabstop>
  <tabstop>Day_radioButton</tabstop>
  <tabstop>OK_pushButton</tabstop>
  <tabstop>Cancel_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
